### PR TITLE
Don't update transaction date when importing manually

### PR DESCRIPTION
**Pull request by Wizmaster**
Following work done in #1559 and update in #2534 (discussion https://github.com/actualbudget/actual/pull/2534#discussion_r1545649961), I think manual import should not override the transaction date as is done with bank account sync.

### CODE REVIEW

**Comment by twk3** [comment]
@Wizmaster what do you think about making this conditional on the cleared status? 

That way uncleared items still get updated with the import, but you can manually edit dates for imported records and they won't be overwritten on further imports?

**Review by MatissJanis — APPROVED** [review]
I've been trying to think of any scenario where this change might not be favourable, but can't come up with anything. So IMO this is good to go.

But still I'd prefer another maintainer to review this. Just as a double-check.

**Comment by Wizmaster** [comment]
@twk3 I think that if a transaction exists before import (manual or bank sync), the inputted date should be kept. I know that sometimes banks can use a later date due to batched transactions not always sent in time, it would be better to always keep the date the end-user inputted.

@MatissJanis 👍

**Comment by youngcw** [comment]
I cant think of a reason to have bank sync act differently than a file import.  In theory they should be the same, just one is automated.  Ill merge this is for now and we will see if any edge users complain about the change.

**Pull request merged**
